examples/ts/get-maximum-spendable.ts

```diff
@@ -10,23 +10,25 @@
 
  * Copyright 2018, BitGo, Inc.  All Rights Reserved.
  */
-import { BitGo } from 'bitgo';
-const bitgo = new BitGo({ env: 'test' });
+import { BitGoAPI } from '@bitgo/sdk-api';
+import { Tbtc } from '@bitgo/sdk-coin-btc';
+require('dotenv').config({ path: '../../.env' });
 
-// TODO: set your access token here
-// You can get this from User Settings > Developer Options > Add Access Token
-const accessToken = '';
-
-// TODO: get the wallet with this id
-const id = '';
+const bitgo = new BitGoAPI({
+  accessToken: process.env.TESTNET_ACCESS_TOKEN,
+  env: 'test',
+});
 
 const coin = 'tbtc';
+bitgo.register(coin, Tbtc.createInstance);
```

Comment:
NIT: I was just wondering, we already is loading the register function with an instance of the chain we want to interact and the instance has the name of the chain in the getChain method. May be we could simplify this registration to remove passing coin name altogether?, Seems redundant to me unless there is a specific reason for this